basic/exercise2/script2.js

```diff
@@ -1,10 +1,23 @@
 'use strict';
 
-var side1 = 5;
-var side2 = 6;
-var side3 = 7
-var trianglePerimeter = (side1 + side2 + side3) / 2;
-var triangleArea = Math.sqrt(trianglePerimeter * ((trianglePerimeter - side1) * (trianglePerimeter - side2) * (trianglePerimeter - side3)));
-document.getElementById("triangle").innerHTML = triangleArea;
+function findingTriangleArea(side1, side2, side3) {
+    if (!(side1 > 0)) {
+        throw new Error("side1 should be a longer than 0");
+    };
+    if (!(side2 > 0)) {
+        throw new Error("side2 should be a longer than 0");
+    };
+    if (!(side3 > 0)) {
```

Comment:
Can't if be simply:
if (side3<=0)
?
It express the same, but is easier to understand